CMGTools/TTHAnalysis/macros/fractionCutTree.py

```diff
@@ -0,0 +1,20 @@
+import ROOT
+import os, sys
+
+# usage: python fractionCutTree.py BIGTREE_DIR FTREE_DIR CUT_STRING DATASET_NAME ...
```

Comment:
Can you add one extra comment line explaining what is this macro supposed to do (and possibly also some of the others)?